pkg/controller/account/iam.go

```diff
@@ -596,13 +467,13 @@ func (r *ReconcileAccount) BuildIAMUser(reqLogger logr.Logger, awsClient awsclie
 	secretName := account.Name
 
 	// Append to secret name if we're if its the SRE admin user secret
-	if iamUserName == iamUserNameSRE {
-		secretName = account.Name + "-" + strings.ToLower(iamUserName)
+	if strings.Contains(iamUserNameSRE, iamUserName) {
+		secretName = account.Name + "-" + strings.ToLower("osdManagedAdminSRE")
```

Comment:
I think we should still have the base names `osdManagedAdmin` and `osdManagedAdminSRE` in the `shared_types` file.

Reply:
So I'm not doing that here because of the way the secret is created, I don't want the `osd-creds-mgmt-asdfx-osdmanagedadminsre-secret` for instance to look like `osd-creds-mgmt-asdfx-osdmanagedadminsre-asdfx-secret`. We would create a card to decouple the secret creation process here?

Reply:
I'm saying just make
var OsdManagedAdminSRE = "osdManagedAdminSRE" and use that.